### ISSUE TITLE
ffmpeg failed to concatenate files

### ISSUE THREAD

**Issue by BBmine** [unfa/kdenlive-multirender#4]
[concat @ 0x5589597ea3c0] Impossible to open 'ST1-Story_Mode-Pt1-01.mp4'
list.txt: No such file or directory

The video files are saved to somewhere other than kdenlive-mulitirender.sh and the kdenlive script.

**Comment by BBmine** [comment]
Okay, so I found out that if the video destination is the same directory as the scripts, it works fine.

**Comment by unfa** [comment]
yeah, currently I only tested the script by putting it to the same directory that your source script and render output is in.
